### PR TITLE
add keys to trigger active abilities

### PR DESCRIPTION
**Pull request by Skiftcha**
e.g. for mantle of the raven

### CODE REVIEW

**Comment by Skiftcha** [comment]
maybe i will think about better solution, e.g. do not touch active ability wheel logic but add separate key shortcuts for first 1..n abilities.
i just want to share some useful feature i made for myself.

**Comment by Dream-Master** [comment]
@Skiftcha so is this a change we need to add or you not done yet and i mark it as draft?

**Comment by Skiftcha** [comment]
@Dream-Master now ready to add

File: src/main/java/travellersgear/client/KeyHandler.java [hunk 1]
```diff
@@ -54,7 +63,11 @@ public void playerTick(TickEvent.PlayerTickEvent event)
 			else if(keyDown[0])
 				keyDown[0] = false;
 
-			if(activeAbilitiesWheel!=null && activeAbilitiesWheel.getIsKeyPressed() && !keyDown[1] && ActiveAbilityHandler.instance.buildActiveAbilityList(player).length>0)
+			Object[][] gear = ActiveAbilityHandler.instance.buildActiveAbilityList(player);
```
Comment (Skiftcha) [inline src/main/java/travellersgear/client/KeyHandler.java:66]: i don't like it is not lazy calculated now

**Comment by Skiftcha** [comment]
since there may be more than 3 active ability items and/or their order in active ability list is not well defined i added command to bind hotkeys to items. so you hold item in hand, type /travellersgear bind 1 and active ability key 1 will trigger this item ability.
this works but i am not very experiences in modding and this need to be reviewed

**Comment by Dream-Master** [comment]
@Skiftcha ae you done with the pr or more stuff will follow ?

**Comment by Skiftcha** [comment]
done

**Comment by Dream-Master** [comment]
@repo-alt @botn365 @bartimaeusnek @Glease @glowredman can you do a review. Thanks

**Review by glowredman — APPROVED** [review]

**Comment by Dream-Master** [comment]
@Skiftcha can recheck please

**Comment by Dream-Master** [comment]
SkiftchaToday at 2:38 PM
i dont think it is too hard, but i can make another pr for this
DreamMasterXXLToday at 2:39 PM
ok i merge this now and we get a second one later. great :slight_smile:
SkiftchaToday at 2:39 PM

**Pull request merged**